packages/frontend/src/components/EditorSettings/Notifications.tsx

```diff
@@ -28,9 +28,10 @@ const defaultFrequency = Frequency.Once
 
 export default function Notifications() {
   const { flowId } = useParams()
-  const [frequency, setFrequency] = useState(defaultFrequency)
   const { data, loading } = useQuery(GET_FLOW, { variables: { id: flowId } })
   const flow: IFlow = data?.getFlow
+  const frequency =
+    flow?.config?.errorConfig?.notificationFrequency ?? defaultFrequency
```

Comment:
nit: convention to have constants in all caps - e.g. `DEFAULT_FREQUENCY`. Alternatively you can just put `Frequency.Once` because the intent is pretty clear that it's the default.